frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1364,6 +1425,31 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
+	/// Remove the current solutions and proceed to the following round.
+	fn do_force_rotate_round() {
+		if Self::current_phase().is_signed() {
+			Self::finalize_signed_phase();
```

Comment:
There's no point in doing this, which is rather expensive, if we are going to throw the solution away. We should return everyone's deposit, and not verify anything.